packages/payment-integration-api/src/cart/carts.mock.ts

```diff
@@ -0,0 +1,41 @@
+import { Cart, CartSource } from '../cart';
+import { getCoupon } from '../coupon/coupons.mock';
+import { getCurrency } from '../currency/currency.mock';
+import { getDiscount } from '../discount/discounts.mock';
+
+import {
+    getCustomItem,
+    getDigitalItem,
+    getGiftCertificateItem,
+    getPhysicalItem,
+} from './line-items.mock';
+
+export default function getCart(): Cart {
```

Comment:
Take a look at the `payment-integrations-test-utils` package. We already have this mock there.

And review all the other files pls. We need to avoid duplication and keep everything in one place

Reply:
@bc-nick @vitalii-koshovyi I saw that those mocks are in `payment-integrations-test-utils` but unfortunately I couldn't use it as it creates circular dependency. 
![Screenshot 2024-10-10 at 15 03 31](https://github.com/user-attachments/assets/0db7db58-485c-45de-9310-9d07568c28a9)

I saw that in the past there were some duplicated mock added to `payment-integrtaion-api`, probably to overcome the same issue. I know it is not perfect solution but I am not sure if we have alternative option. What do you think?

Reply:
@serhii-tkachenko @PavlenkoM do you have better ideas?

Reply:
I believe we need to consider future refactoring and move mocks to `payment-integrtaion-api` package.|
@animesh1987 @bc-peng FYI

Reply:
I don't have better idea.
Previously I had similar problem and discussed this with @bigcommerce/team-checkout , we decided that only solution to duplicate mocks.

Reply:
In this case it might be better to get rid of `payment-integrations-test-utils` and move them to `payment-integration-api` 👀

Reply:
Yes, I agree that it might be good for us to consider removing `test-utils` in order to avoid the circular dependencies   whenever we need to test anything requiring a cart mock in the `api`.